yt_dlp/extractor/tokentube.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import functools
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    parse_count,
+    unified_strdate,
+    js_to_json,
+    OnDemandPagedList,
+)
+
+
+class TokentubeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?tokentube\.net/(?:view)?/?\??[lv]=?/?(?P<id>\d+)(?:/.*)?'
+    _TESTS = [{
+        'url': 'https://tokentube.net/l/3236632011/Praise-A-Thon-Pastori-Chrisin-ja-Pastori-Bennyn-kanssa-27-8-2021',
+        'info_dict': {
+            'id': '3236632011',
+            'ext': 'mp4',
+            'title': 'Praise-A-Thon Pastori Chrisin ja Pastori Bennyn kanssa 27.8.2021',
+            'description': '',
```

Comment:
Is the description actually empty, or we just cant find it?

Reply:
Description is empty.